scripts/obtain/dotnet-install.sh

```diff
@@ -656,7 +656,9 @@ install_dotnet() {
     say_verbose "Zip path: $zip_path"
 
     say "Downloading link: $download_link"
-    download "$download_link" $zip_path || download_failed=true
+    # Failures are expected in the non-legacy case. Do not output to stderr, since
```

Comment:
@johnbeisner make sure to review this part.

Reply:
I discussed this line with him before